### ISSUE TITLE
[6.x] PHPDoc issue

### ISSUE THREAD

**Issue by riccardobosini** [cknow/laravel-money#130]
Good morning, 
I'm writing cause I've the following issue.
I've a project with installed this package.
Let's suppose this code: 
```php
use Money\Currency;

$revenueA = new \Cknow\Money\Money(123, new Currency('EUR'));
$revenueB = new \Cknow\Money\Money(123, new Currency('EUR'));

$revenueTotal = $revenueA->add($revenueB);
```
In my case PHPStorm highlight the last row showing this message: `Expected parameter of type '\Money\Money', '\Cknow\Money\Money|null' provided`. It expects that the parameter passed to the `add` method to be of type `\Money\Money`.

It's annoying cause working with amounts the screen is full of warnings.

Can you please help me about this?
Thanks in advance.

**Comment by ricardogobbosouza** [comment]
Hi @riccardobosini 
It is not what is in https://github.com/cknow/laravel-money/blob/main/src/Money.php#L24
What version of `cknow/laravel-money` are you using?

**Comment by riccardobosini** [comment]
> Hi @riccardobosini It is not what is in https://github.com/cknow/laravel-money/blob/main/src/Money.php#L24 What version of `cknow/laravel-money` are you using?

I do not think so. The warning shows up doing every operation on an object of class `\Cknow\Money\Money`.

![image](https://user-images.githubusercontent.com/23011509/205269787-e2c559da-a93e-4015-9fbe-7cd00ccb420f.png)

**Comment by ricardogobbosouza** [comment]
It was fixed in [v7.0.0](https://github.com/cknow/laravel-money/releases/tag/v7.0.0)

**Comment by ricardogobbosouza** [comment]
I'll fix it in v6 too

**Comment by ricardogobbosouza** [comment]
@riccardobosini 
Fixed [v6.4.1](https://github.com/cknow/laravel-money/releases/tag/v6.4.1)

**Comment by ricardogobbosouza** [comment]
Recommended to use the latest version v7
